### DISCUSSION TITLE
How do I compile my AT project?

### DISCUSSION THREAD

**Discussion by micronik-ir** [JAndrassy/WiFiEspAT#73] [General]
Hello
I used the AT 2 firmware for ESP8266-01 module and the MQTT protocol worked fine. But in this firmware, **"HTTP AT Command"** protocol and **"Web server AT Command"** are disabled by default. I tried to enable these features but failed. Please release the firmware that supports these features for the ESP8266 module.
Thanks

**Comment by JAndrassy** [comment]
I compiled it with the default configuration by Espressif for 1 MB esp8266. I didn't change any settings. I guess it is disabled because there is no space on flash for it.
